src/components/Employees/EmployeesTable/HRmodalWindows/HRmodalWindow.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import '../../../../styles/modal.scss';
+import CloseIcon from '@material-ui/icons/Close';
+import {IconButton, FormControl, InputLabel, MenuItem, Select} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import './HRmodalWindow.scss';
+import Button from '@material-ui/core/Button';
+
+export const HRmodalWindow = ({name, handleClose}) => {
+
+  const englishLevel = ['Beginner(A1)', 'Elementary(A2)', 'Intermediate(B1)', 'Upper Intermediate(B2)', 'Advanced(C1)', 'Proficiency(C2)'];
+  const priority = ['low', 'high'];
+
+  let itemKey = 0;
+  const modalBody = <>
+    <div className='assign-level'>You want to assign a test for {name}</div>
```

Comment:
Add Paper wrapper with property of elevation={2}. Because your modal window doesn't support dark theme